validator/project_validator.go

```diff
@@ -116,6 +119,16 @@ func (v ValidationErrors) HasError() bool {
 	return false
 }
```

Comment:
Can we remove this and replace with `Has(Error)`? It's duplicate functionality now.

Reply:
Yes! I actually removed all instances already and meant to delete this- ty for the catch